instrumentation/rack/example/trace_demonstration.rb

```diff
@@ -18,7 +18,7 @@
 builder = Rack::Builder.app do
   # integration should be automatic in web frameworks (like rails),
   # but for a plain Rack application, enable it in your config.ru, e.g.,
-  use OpenTelemetry::Instrumentation::Rack::Middlewares::TracerMiddleware
+  use ::Rack::Events, [OpenTelemetry::Instrumentation::Rack::Middlewares::EventHandler.new]
 
```

Comment:
Since using the `TracerMiddleware` is still a configurable option, do you think it's valuable to mention it in one of the Rack examples?

Reply:
Sure I could add more. I think the follow up PR would be to remove the middleware implementation entirely.

Reply:
Got it. Let's leave it as-is then.